src/main/scala/io/kf/etl/processors/participantcommon/transform/step/WriteJsonSink.scala

```diff
@@ -8,8 +8,7 @@ object WriteJsonSink {
   def apply[T](processorName: String)(step: String)(input: Dataset[T])(implicit config: Config, spark: SparkSession): Dataset[T] = {
```

Comment:
I would verify if this class is already used. If not, I would rather implement export to jsin directly into JsonOutputProcessor and delete this class